code/ui/components/src/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Badge } from './Badge';
+
+export default {
+  title: 'Basics/Badge',
```

Comment:
We should move towards using `autotitle` for all our stories (not necessarily in this PR)

Reply:
As you probably noticed I did this in #19615 where possible.